website/docs/r/app_service_source_control_token.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_app_service_source_control_token"
+sidebar_current: "docs-azurerm-resource-app-service-source-control-token"
+description: |-
+  Manages an App Service source control token.
+
+---
+
+# azurerm_app_service_source_control_token
+
+Manages an App Service source control token.
+
+~> **NOTE:** A source control token is global and required only once per Azure account.
```

Comment:
would this be clearer as:

```suggestion
~> **NOTE:** Source Control Token's are configured at the subscription level, not on each App Service - as such this can only be configured Subscription-wide
```